tests/conftest.py

```diff
@@ -632,7 +634,47 @@ def product(product_type, category):
 
 
 @pytest.fixture
-def product_with_two_variants(color_attribute, size_attribute, category):
+def product_with_single_variant(product_type, category):
+    product = Product.objects.create(
+        name="Test product with single variant",
+        price=Money("1.99", "USD"),
+        product_type=product_type,
+        category=category,
+        is_published=True,
+    )
+    variant = ProductVariant.objects.create(
+        product=product,
+        sku="SKU_SINGLE_VARIANT",
+        cost_price=Money("1.00", "USD"),
+        quantity=101,
+        quantity_allocated=1,
+    )
+    return product
+
+
+@pytest.fixture
+def product_with_two_variants(product_type, category):
+    product = Product.objects.create(
+        name="Test product with two variants",
+        price=Money("10.00", "USD"),
+        product_type=product_type,
+        category=category,
+    )
+
+    for i in (1, 2):
+        ProductVariant.objects.create(
+            product=product,
+            sku=f"prodVar{i}",
+            cost_price=Money("1.00", "USD"),
+            quantity=10,
+            quantity_allocated=1,
+        )
```

Comment:
We should keep the number of DB queries as small as possible. Especially for fixtures.
https://docs.djangoproject.com/en/2.2/ref/models/querysets/#bulk-create